coordinator/src/services/communication/sandbox-spine-client.ts

```diff
@@ -6,9 +6,37 @@ import {
 } from "@models"
 import {SpineClient} from "./spine-client"
 import {StatusCheckResponse} from "../../utils/status"
+import {isTrackerRequest} from "../../../../models/spine"
 
 export class SandboxSpineClient implements SpineClient {
-  async send(spineRequest: spine.SpineRequest): Promise<spine.SpineResponse<unknown>> {
+  async send(clientRequest: spine.ClientRequest): Promise<spine.SpineResponse<unknown>> {
+    if(isTrackerRequest(clientRequest)) {
+      return await this.handleTrackerRequest()
+    } else {
+      return await this.handleSpineRequest(clientRequest)
+    }
+  }
+
+  async poll(): Promise<spine.SpineResponse<fhir.OperationOutcome>> {
+    return Promise.resolve({
+      statusCode: 400,
+      body: notSupportedOperationOutcome
+    })
+  }
+
+  async getStatus(): Promise<StatusCheckResponse> {
+    return {
+      status: "pass",
+      timeout: "false",
+      responseCode: 200
+    }
+  }
+
+  async handleTrackerRequest(): Promise<spine.SpineResponse<unknown>> {
+    return Promise.resolve({body: "Not Supported", statusCode: 400})
```

Comment:
should we match the object on lines 21-24?

Reply:
Done ✔️